public/app/plugins/datasource/cloudwatch/components/VariableQueryEditor/VariableQueryEditor.tsx

```diff
@@ -163,14 +164,20 @@ export const VariableQueryEditor = ({ query, datasource, onChange }: Props) => {
             placeholder="attribute name"
             onBlur={(value: string) => onQueryChange({ ...parsedQuery, attributeName: value })}
             label="Attribute Name"
+            tooltip='Attribute or tag to query on. Tags should be formatted "Tags.<name>".'
```

Comment:
what about adding a link to the documentation so people know what they can use here?